pkg/operator/sync.go

```diff
@@ -37,14 +37,28 @@ func (optr *Operator) syncAll(config *OperatorConfig) error {
 			glog.Errorf("Error syncing cluster api controller: %v", err)
 			return err
 		}
-		glog.V(3).Info("Synced up all components")
+		glog.V(3).Info("Synced up all cluster api controller components")
 	}
-
 	if err := optr.statusAvailable(); err != nil {
 		glog.Errorf("Error syncing ClusterOperatorStatus: %v", err)
 		return fmt.Errorf("error syncing ClusterOperatorStatus: %v", err)
 	}
 
+        // In addition, if the Provider is BareMetal, then bring up
+        // the baremetal-operator pod
+        if config.BaremetalPodConfig.BaremetalOperator != "" {
```

Comment:
config.BaremetalPodConfig != nil would be better here.

Reply:
Yes, that is better.